apps/google-analytics-4/frontend/src/components/config-screen/api-access/setup/SetupServiceAccountCard.tsx

```diff
@@ -155,7 +155,8 @@ export default function SetupServiceAccountCard(props: Props) {
           rel="noopener noreferrer">
           Google Cloud service account
         </TextLink>{' '}
-        for which you enable <i>read access</i> to your organization's Google Analytics data.
+        for which you enable "viewer" access to one of your organization's Google Analytics
+        properties.
```

Comment:
i recommend:

```
The Google Analytics 4 app requires a
[Google Cloud service account](https://www.contentful.com/help/google-analytics-service-account-setup/)
 with "viewer" access to one or more Google Analytics properties.
```

Reply:
I like it 👍

Reply:
Going with your suggestion with a few minor modifications:

> The Google Analytics 4 app requires a [Google Cloud service account] with "viewer" access to one of your organization's Google Analytics 4 properties.